### PR TITLE
[Docs] - Added New FAQ

### LINKED ISSUE #3511: Create FAQ section in Meshery contributing Docs
#### Description

Currently, Meshery Doc does not have a properly compiled FAQ and answers

#### Expected Behavior

- You are expected to add more FAQ's and answers to the FAQ section on the doc
-  Move the FAQ to https://docs.meshery.io/project/contributing
-  Ensure there is consistency in questions and answers format.

#### Screenshots/Logs

Screenshot 2021-07-14 at 20 44 06
#### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker or Kubernetes

---

**Contributor Resources:**
- (Contributing Docs](https://docs.meshery.io/project/contributing)

### PR DESCRIPTION
**Pull request by The-Loony-Coder**
Signed-off-by: Mritunjay <msengar121@gmail.com>

**Description**

This PR Fixes #3511

**Notes for Reviewers**
Have a look at the links that I've provided, should I include them or not, or should I change them.
I followed the FAQ pattern from [adapters](https://docs.meshery.io/concepts/architecture/adapters) page(faq are at the bottom).

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9460101fd6ff8d85482f61548e70f75c3b18a137 at: https://61123f8aedccb7ea10db5f22--meshery-docs-preview.netlify.app

**Comment by warunicorn19** [comment]
I think we should add some sort of bullets or numbers to differentiate between questions and answers, right now it all
 looks really similar.

**Comment by The-Loony-Coder** [comment]
Are there any specific guidelines or design patterns that the documentation is following or should we decide our own.
Should we go with bullets,square bullets or numbers,or should I write "question"  in front of question.
Or we should just number them.

**Comment by warunicorn19** [comment]
There arent actually, lets hold off till @adithyaakrishna or @leecalcote review this.

**Comment by adithyaakrishna** [comment]
@Anita-ihuman Could you please check the new FAQs which @The-Loony-Coder has added and suggest any changes if necessary :)

**Comment by Anita-ihuman** [comment]
@The-Loony-Coder great job providing the answers for these questions.
I learnt a lot from them.

Concerning the pattern,
https://layer5.io/company/faq This is what most of our FAQ’s look like.
If you would like to pick inspiration from this.
Other than that, you did a fantastic job

File: docs/pages/project/FAQ.md [hunk 1]
```diff
@@ -6,16 +6,42 @@ description: General commonly asked questions and answers about Meshery.
...
 
-## Question: Why was Meshery created?
+####  Why was Meshery created?
 
-**Answer**: _As an open source, vendor neutral project, Meshery was created out the necessity to enable service mesh adopters to overcome the challenge of complex virtual networking; to come to expect more from their infrastructure; to enable the world to understand and operate any service mesh with confidence._
+ As an open source, vendor neutral project, Meshery was created out the necessity to enable service mesh adopters to overcome the challenge of complex virtual networking; to come to expect more from their infrastructure; to enable the world to understand and operate any service mesh with confidence.
 
-<br />
-<div class="center" style="color:gray;position:relative;top:-10px;font-size:1.25em;">Don't find an answer to your question here? Ask in the <a href="https://layer5io.slack.com/archives/C01AFD2D547">#support</a> channel in the <a hre="http://slack.layer5.io/">Layer5 Slack</a>.</div>
+####  What does meshery do?
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:18]: Always capitalize Meshery.

File: docs/pages/project/FAQ.md [hunk 2]
```diff
@@ -6,16 +6,42 @@ description: General commonly asked questions and answers about Meshery.
...
+
+ - Operates service meshes with confidence
+   - with operational best practices. 
+ - Compare apples-to-apples performance across service meshes.
+ - Understand behavioral differences between service meshes.
+ - Track your application performance from version to version.
+
+#### What is mesheryctl?
+
+A Command Line Interface to manage Meshery.
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:28]: command line interface can be lowercased.

File: docs/pages/project/FAQ.md [hunk 3]
```diff
@@ -6,16 +6,42 @@ description: General commonly asked questions and answers about Meshery.
...
+
+#### How to run Meshery?
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:30]: Improper grammar. Please rephrase.

File: docs/pages/project/FAQ.md [hunk 4]
```diff
@@ -6,16 +6,42 @@ description: General commonly asked questions and answers about Meshery.
...
+- [Scoop user](https://github.com/meshery/scoop-bucket) 
 
-**Answer**: _Yes, both of them do run Meshery on your local machine. `make run local` builds Meshery from source and runs it on your local OS, while `mesheryctl system start` runs Meshery as a set of containers in Docker on your local machine._
+
+#### What is the difference between `make run-local` and `mesheryctl system start`? Do they both run Meshery on my local machine?
+
+  Yes, both of them do run Meshery on your local machine. `make run local` builds Meshery from source and runs it on your local OS, while `mesheryctl system start` runs Meshery as a set of containers in Docker on your local machine.
+
+ <br />
+<div class="center" style="color:gray;position:relative;top:-10px;font-size:1.25em;">Don't find an answer to your question here? Ask in the <a href="https://layer5io.slack.com/archives/C01AFD2D547">#support</a> channel in the <a hre="http://slack.layer5.io/">Layer5 Slack</a>.</div>
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:45]: Missing the `f` in `href`

File: docs/pages/project/FAQ.md [hunk 5]
```diff
@@ -6,16 +6,42 @@ description: General commonly asked questions and answers about Meshery.
...
+- Bash user → Meshery Installation [Quick Start](https://docs.meshery.io/installation/quick-start).
+- [Brew user](https://github.com/meshery/homebrew-tap)
+- Kubernetes user
+- [Scoop user](https://github.com/meshery/scoop-bucket) 
 
-**Answer**: _Yes, both of them do run Meshery on your local machine. `make run local` builds Meshery from source and runs it on your local OS, while `mesheryctl system start` runs Meshery as a set of containers in Docker on your local machine._
+
+#### What is the difference between `make run-local` and `mesheryctl system start`? Do they both run Meshery on my local machine?
+
+  Yes, both of them do run Meshery on your local machine. `make run local` builds Meshery from source and runs it on your local OS, while `mesheryctl system start` runs Meshery as a set of containers in Docker on your local machine.
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:42]: ...as a set of containers in Docker or in Kubernetes...

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Regarding formatting, yes, please leave the italics for answers and the labels "Question:" and "Answer:" as denotations.

**Comment by The-Loony-Coder** [comment]
> Regarding formatting, yes, please leave the italics for answers and the labels "Question:" and "Answer:" as denotations.

should I also change the formatting [here](https://docs.meshery.io/concepts/architecture/adapters) "FAQ of adapters page". also.

**Comment by l5io** [comment]
🚀 Preview for commit 2476b1bf873ccc289207223c9cc1af5fe60aeb77 at: https://611bcf86c8b22c3e60b25a7d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0aad2f906fa3a2343dd954ce39c1e9d6c01179bd at: https://611bd0a00955e93ee9f2ac83--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4e7646dae5008e8cb69b2684adb9268f3b276f97 at: https://611bf3cf9fa1e315478c06b3--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8cca93b54fe5944aa303caaefa58db183d9fcacc at: https://611bf54265309c00a3bf9769--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5e50975965bbde47fbeb53d38316270ab25c32fd at: https://611c9bed1577d5a201799b0c--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2abebef851594bce04198c90d755f6578f5c5d61 at: https://611ca0261936509c6a864503--meshery-docs-preview.netlify.app

**Pull request merged**